docs/customizations/node-exporter-namespace.md

```diff
@@ -0,0 +1,13 @@
+### node-exporter DaemonSet namespace
+
+Standard Kubernetes manifests are all written using [ksonnet-lib](https://github.com/ksonnet/ksonnet-lib/), so they can be modified with the mixins supplied by ksonnet-lib. For example to override the namespace of the node-exporter DaemonSet:
+
+```jsonnet mdox-exec="cat examples/ksonnet-example.jsonnet"
+((import 'kube-prometheus/main.libsonnet') + {
+   values+:: {
+     nodeExporter+: {
+       namespace: 'my-custom-namespace',
+     },
+   },
+ }).nodeExporter.daemonset
+```
```

Comment:
Some examples are pretty outdated or don't add much value, should we keep those?

Reply:
let's remove such examples.